sdk/loadtestservice/loadtestservice/.eslintrc.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
why do we have both sdk/loadtestservice/loadtesting-rest and sdk/loadtestservice/loadtestservice and both of them are rlc and look almost the same ?

Reply:
I am not sure why it was still showing old "loadtestservice" folder, have removed it.